src/gui/DatabaseOpenWidget.cpp

```diff
@@ -137,13 +129,12 @@ void DatabaseOpenWidget::load(const QString& filename)
 {
     m_filename = filename;
 
-    m_ui->labelFilename->setText(filename);
-
+    m_ui->comboKeyFile->addItem(tr("None"), -1);
```

Comment:
To align with the hardware key, recommend "Select file..."

Reply:
I had that before, but wasn't sure if it's clear enough that you have to select that in order to get rid of a key file. Will do, though.

Reply:
hmmm good point. Perhaps you can add the "clear" button to the right similar to the search box. When clicked it goes back to "select file...".

Reply:
That would indeed be an option.